portality/static/widget/public_search_config.js

```diff
@@ -0,0 +1,6 @@
+$.extend(true, doaj, {
+    publicSearchConfig : {
+        publicSearchPath : 'http://localhost:5004/query/journal,article/_search?ref=fqw',
```

Comment:
Is this just for developers, or do we need to update this before finalising for release?

Reply:
that is the error. It has to be changed to `doaj_url + '\query\journal,article\_search?ref=fqw'`